.vscode/cSpell.json

```diff
@@ -2,20 +2,38 @@
 {
   // version of the setting file
   "version": "0.2",
-
   // spelling language
   "language": "en",
-
   // custom dictionaries
-  "dictionaries": ["acronyms+names", "jargon", "people", "shell", "workspace"],
+  "dictionaries": [
```

Comment:
please move the unrelated changes into a different PR

Reply:
the format and sorting changes in the subfolder ".vscode" where done by vscode itself. So I was assuming that this is fine. If you don't accept that, this means for me that I have to do the changes there manually which is inconvenient. I guess I'm not the only person with this issue, am I? How do you do the changes there?